packages/expo-blur/src/index.ts

```diff
@@ -1,2 +1,2 @@
 export { default as BlurView } from './BlurView';
-export { BlurTint, BlurProps } from './BlurView.types';
+export { BlurTint, BlurViewProps } from './BlurView.types';
```

Comment:
Isn't it a breaking change if someone imported this type? 🤔

Reply:
Thank you for catching this! I intended to add this change to the CHANGELOG, but forgot about this later on 😅